### PR TITLE
remove pre-release increment constraint

### PR DESCRIPTION
**Pull request by lfaoro**
I don't see a reason why pre-releases should not be incremented. It is very common to release early without a stable api.

We should support incrementing 0.1.0 releases and the like.
#59

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Binaries for programs and plugins
```
Comment (hanzei) [inline .gitignore]: Please remove these unrelated changes

Comment (lfaoro) [reply .gitignore]: I did so by reverting the gitignore commit, sorry for pushing unrelated changes.

**Comment by lfaoro** [comment]
@hanzei would be possible to get this merged?

**Comment by hanzei** [comment]
I don't have write access to this repo.

**Comment by lfaoro** [comment]
That’s sad 😕

@hanzei do you know any anyone that has? You could tag them so they get a notification.

**Comment by hanzei** [comment]
@blang Can you please take a look at this PR?

**Comment by ltpquang** [comment]
hi there, any progress on this PR so far? i got stuck at this point too!

**Comment by lfaoro** [comment]
hi @ltpquang -- beside submitting the commits and updating the tests, I don't know what else should I do. 3 months later, no answer from the maintainers.

For now I just forked the repo and I'm using my fork with the constraint removed: https://github.com/lfaoro/semver/commits/master

**Comment by blang** [comment]
Rebased and pushed to a64359f. Thanks for your contribution and long wait!
